data/heliarch intro.txt

```diff
@@ -0,0 +1,1570 @@
+	on complete
+		conversation
+			`Immediately upon landing, Iplora is the first to leave your ship, disappearing into the spaceport before you could stop her. You ask Tugroob and Kirrie about it, and they say that she is just taking a small break.`
+			`	"Value moments of solitude, the Saryd do. To a calm oasis, she will head for the day," Kirrie explains, as he also prepares to leave, claiming to know some Heliarchs stationed here who could help him look into some rumors.`
+			`	Being the last Heliarch agent in your ship, Tugroob turns to you, and asks, "Assist me in my search, would you, Captain?"`
+			choice
+				`	"Of course, where are we headed?"`
+					goto desertspider
+				`	"Sorry, I'll have to refuse this time."`
+					goto lonespider
+			label desertspider
+			`	"Reports of suspicious transactions, we have received. An outdated, abandoned warehouse, the place is. For storage of the few crops that can grow here, used, it was," they say as you put in the coordinates they give you.`
+			`	You follow Tugroob to the storage area after landing, spotting a desert storm brewing in the distance as you make your way there. A large crack has formed, around the area where they said the crops were originally planted and tested.`
+			`	The barn itself is quite large and could probably hold more stockpile than one of the medium-sized Arach freighters in it's earlier days, but now many of the planks making up the walls have been torn down in several spots.`
+			choice
+				`	"Let's check out the barn."`
+					goto barn
+				`	"Maybe we should enter that hole where the crops used to be."`
+					goto holeintheground
+			label barn
+			`	The two of you head inside to find the place nearly entirely empty, save for a few sets of crates and barrels around the corners.`
+			`	You follow Tugroob to the upper levels, but once again find nothing but dusty containers. A small gap between the planks here allows some sand to come inside, the storm is getting closer.`
+			`	"Much too obvious a hiding spot, this would be," they say.`
+			choice
+				`	"You're right, should we go outside and check around that hole?"`
+					goto anotherholeintheground
+				`	"Maybe, but that could also make someone consider it an even better one. Let's open up these crates."`
+					goto opencrate
+			label opencrate
+			`	You two go about opening all the crates on the upper floor, but end up finding nothing.`
+			`	On the bright side, you manage to use the remains to patch up the gap, as the storm grew stronger.`
+			`	"Nothing here, we will find, Captain," Tugroob says.`
+			choice
+				`	"Yeah, sorry, let's go outside and check that hole while we can."`
+					goto aslipperyholeintheground
+				`	"Hold on, we can still check the ones down there."`
+					goto crateeater
+			label crateeater
+			`	At this point Tugroob starts using all their body weight and gripping the crates with all legs at the same time, crushing each crate and barrel open.`
```

Comment:
```suggestion
			`	At this point, Tugroob starts using all their body weight to grip the containers with all legs at the same time, crushing each crate and barrel open.`
```
Didn't like the double usage of `crates` when it's actually referring to both `crates` and `barrels`.